src/main/java/net/sf/jabref/gui/shared/MigrationHelpDialog.java

```diff
@@ -0,0 +1,80 @@
+package net.sf.jabref.gui.shared;
+
+import java.awt.BorderLayout;
+import java.awt.Component;
+import java.awt.Dimension;
+import java.awt.event.ActionEvent;
+import java.awt.event.KeyEvent;
+
+import javax.swing.AbstractAction;
+import javax.swing.Action;
+import javax.swing.Box;
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JComponent;
+import javax.swing.JDialog;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.KeyStroke;
+import javax.swing.border.EmptyBorder;
+
+import net.sf.jabref.gui.help.HelpAction;
+import net.sf.jabref.logic.help.HelpFile;
+import net.sf.jabref.logic.l10n.Localization;
+
+public class MigrationHelpDialog extends JDialog {
+
+    public MigrationHelpDialog(OpenSharedDatabaseDialog openSharedDatabaseDialog) {
+        super(openSharedDatabaseDialog, Localization.lang("Migration help information"));
+        setModal(true);
+
+        String migrationMessage = Localization
+                .lang("Entered database has obsolete structure and is no longer supported.");
+        JLabel migrationLabel = new JLabel(migrationMessage);
+        migrationLabel.setAlignmentX(Component.LEFT_ALIGNMENT);
+
+        String helpMessage = Localization.lang("Click here to learn about the migration of pre-3.6 databases.");
```

Comment:
Since the db structure did only change with v 3.6, you can always write, that the error occured because of a database with the structure used in JabRef with a version < 3.6. It will always be true and does not need an updated version number.